tests/e2e/scale/noobaa/test_scale_endpoint.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import logging
+from ocs_ci.framework.testlib import MCGTest, scale, skipif_ocs_version
+from ocs_ci.ocs import constants, defaults, ocp, scale_pgsql
+from ocs_ci.utility import utils
+
+log = logging.getLogger(__name__)
+
+option1 = {
+    'create': [('name', 'job1'), ('runtime', '900')],
+    'job1': [('iodepth', '4'), ('rw', 'randrw'), ('bs', '32k'), ('size', '64m'), ('numjobs', '4')]
+    }
+option2 = {
+    'create': [('name', 'job1'), ('name', 'job2'), ('name', 'job3'), ('runtime', '900')],
+    'job1': [('iodepth', '4'), ('rw', 'randrw'), ('bs', '32k'), ('size', '64m'), ('numjobs', '4')],
+    'job2': [('iodepth', '16'), ('rw', 'randrw'), ('bs', '64k'), ('size', '512m'), ('numjobs', '4')],
+    'job3': [('iodepth', '32'), ('rw', 'randrw'), ('bs', '128k'), ('size', '1024m'), ('numjobs', '4')]
+    }
+
+
+@scale
+@skipif_ocs_version('<4.5')
```

Comment:
noobaa endpoints auto scale is going to be supported starting OCS 4.6

Reply:
This feature is available since 4.5, therefore it should work when running it on 4.5 and later.